docs/services-developer-guide.md

```diff
@@ -62,6 +59,16 @@ These resources are powered by:
    controller watches for changes to `CloudMemorystoreInstance` resources and
    calls Google's Cloud Memorystore API to create, update, or delete an instance
    as necessary.
+1. The resource claim scheduling controller. A claim scheduling controller
+   exists for each kind of resource class that could satisfy a resource claim.
+   This controller is unaware of any external system - it simply schedules
+   resource claims to resource classes that match their class selector labels,
+   so that they may be handled by the resource claim controller.
+1. The resource claim defaulting controller. A claim defaulting controller
+   exists for each kind of resource class that could satisfy a resource claim.
+   This controller is unaware of any external system - it allocates resource
+   claims that do not specify a class selector to a resource class annotated as
```

Comment:
Maybe add does not specify a class selector / class reference / managed reference? Not sure if that would just make it more confusing? Potentially solved by the example implementation below which specifies the predicates.

Reply:
I think these comments below https://github.com/crossplaneio/crossplane/pull/973/files#diff-06c88f31ffd6d41fcdf2a50c38c6c9afR803 are sufficient.

Reply:
> I don't think going into detail on how these controllers work from a user's perspective is best explained in the developer guide.

This is another area where I went with the above philosophy.